### PR TITLE
Add net.audiorelay.AudioRelay

### PR DESCRIPTION
**Pull request by cry0ice**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  
Stream audio between your devices.
- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** https://community.audiorelay.net/t/appimage-or-flatpak-suggestion-for-better-distribution-for-linux/838/17?u=cryo

```eml
Message-ID: <9cca735d-9b4e-4ed1-9a0c-0ec5ae2c4267@disroot.org>
Date: Sun, 20 Oct 2024 08:52:35 +0800
MIME-Version: 1.0
User-Agent: Mozilla Thunderbird
To: hello@audiorelay.net
Content-Language: en-US
From: Cryo <cryo@disroot.org>
Subject: About publishing AudioRelay to Flathub
Content-Type: text/plain; charset=UTF-8; format=flowed
Content-Transfer-Encoding: 7bit

I am publishing AudioRelay to Flathub.

This is pull request: https://github.com/flathub/flathub/pull/5738

Please be informed
```

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by cry0ice** [comment]
bot, build net.audiorelay.AudioRelay

File: net.audiorelay.AudioRelay.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+      - type: file
+        path: net.audiorelay.AudioRelay.desktop
+
+      - type: file
+        path: net.audiorelay.AudioRelay.metainfo.xml
+
+      - type: extra-data
+        filename: audiorelay.tar.gz
+        url: https://dl.audiorelay.net/setups/linux/audiorelay-0.27.5.tar.gz
```
Comment (hfiguiere) [inline net.audiorelay.AudioRelay.yaml:47]: so this is a universal binary that runs on both x86_64 and aarch64?

Comment (cry0ice) [reply net.audiorelay.AudioRelay.yaml:48]: No, it only works on x86_64. sorry I forget to add limits

**Comment by cry0ice** [comment]
bot, build net.audiorelay.AudioRelay

**Comment by cry0ice** [comment]
bot, build net.audiorelay.AudioRelay

File: net.audiorelay.AudioRelay.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <launchable type="desktop-id">net.audiorelay.AudioRelay.desktop</launchable>
+
+  <url type="homepage">https://audiorelay.net/</url>
+  <url type="bugtracker">https://community.audiorelay.net/c/bug-reports/</url>
+  <url type="faq">https://docs.audiorelay.net/faq</url>
+  <url type="translate">https://translations.audiorelay.net/</url>
+
+  <description>
```
[the remark this answers is not in the document]
Comment (cry0ice) [reply net.audiorelay.AudioRelay.metainfo.xml:19]: I added more description.

**Comment by cry0ice** [comment]
> > I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. Link:
> 
> Did you try to contact upstream developers?

I sent a email to them.

File: net.audiorelay.AudioRelay.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      - install -Dm644 net.audiorelay.AudioRelay.desktop -t /app/share/applications
+    sources:
+      - type: script
+        commands:
+          - exec /app/extra/bin/AudioRelay "$@"
+        dest-filename: audiorelay.sh
+
+      - type: script
+        commands:
+          - tar -xvf audiorelay.tar.gz
```
Comment (bbhtt) [inline net.audiorelay.AudioRelay.yaml:34]: silence output from tar here, it will leak into flatpak cli when installing.

**Comment by bbhtt** [comment]
> > > I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. Link:
> > 
> > 
> > Did you try to contact upstream developers?
> 
> I sent a email to them.

You can post here too https://community.audiorelay.net/t/appimage-or-flatpak-suggestion-for-better-distribution-for-linux/838/16 looks like they link it from the website.

File: net.audiorelay.AudioRelay.yaml (hunk 3 above)
Comment (cry0ice) [reply net.audiorelay.AudioRelay.yaml]: fixed

**Comment by cry0ice** [comment]
bot, build net.audiorelay.AudioRelay

**Comment by cry0ice** [comment]
> > > > I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. Link:
> > > 
> > > 
> > > Did you try to contact upstream developers?
> > 
> > 
> > I sent a email to them.
> 
> You can post here too https://community.audiorelay.net/t/appimage-or-flatpak-suggestion-for-better-distribution-for-linux/838/16 looks like they link it from the website.

https://community.audiorelay.net/t/appimage-or-flatpak-suggestion-for-better-distribution-for-linux/838/17?u=cryo

**Pull request closed without merging**